examples/litepeer/litepeer.go

```diff
@@ -20,6 +20,7 @@ var (
 	stat        = flag.Bool("stat", false, "Get stat of the last fetch")
 	enableLog   = flag.Bool("logToStderr", false, "Enable app logs on stderr")
 	showProg    = flag.Bool("progress", false, "Enable progress on stdout")
+	jsonOut     = flag.Bool("json", false, "Display output in json format")
 )
 
 func returnError(err string, printUsage bool) {
```

Comment:
We need a man page type usage doc.... we can use this same output to display on the CLI docs web

Reply:
We can have a `-help` flag to `PrintDefaults` usage